sklearn/linear_model/tests/test_sag.py

```diff
@@ -269,7 +270,6 @@ def test_classifier_matching():
         assert_array_almost_equal(intercept2, clf.intercept_, decimal=9)
 
 
-@ignore_warnings
```

Comment:
I really do not get this weird side-effect at a distance kind of thing ... it feels like something is really fishy here. I'll try to reproduce locally.